client/blocks/reader-site-subscription/site-subscription-subheader.tsx

```diff
@@ -67,20 +66,19 @@ const SiteSubscriptionSubheader = ( {
 	const recordViewFeedButtonClicked = useRecordViewFeedButtonClicked();
 
 	subheaderItems.push(
-		<Button
-			key={ `view-feed-button-${ feedId }` }
-			title={ translate( 'View feed' ) }
-			className="site-subscription-header__view-feed-button"
-			icon={ <FeedIcon /> }
+		<a
 			href={ getFeedUrl( feedId ) }
+			title={ translate( 'View feed' ) }
```

Comment:
Since we now have button label, fine to remove `title`

Reply:
fixed 👍